versioned_docs/version-8.2/apis-tools/web-modeler-api/index.md

```diff
@@ -5,7 +5,8 @@ description: "Web Modeler API (beta) is a REST API and provides access to Web Mo
 ---
 
 :::caution Beta Offering
```

Comment:
Should this be removed in 8.2 docs? I see in line 71 you remove the `/beta/`.

Reply:
Reading the PR description I think this is correct as-is, but I just want to confirm.

Reply:
See the PR description:

> `v1` will not be backported to Web Modeler SM 8.2 which is why we will have a hybrid state in the 8.2 docs where for SaaS, we talk about `v1` and for SM, we talk about `beta`.

Do you think it would make sense to explicitly mention this in the 8.2 docs?

Reply:
![image](https://github.com/camunda/camunda-platform-docs/assets/1051363/e840f042-8936-4d30-ac10-9455815aef77)

Yes, since it almost looks like a typo if I didn't read the admonition at the top of the page. I would also recommend adjusting the response to be `v1` so it's consistent with the SaaS experience. Or remove the version line entirely.

If we wanted to get fancy, we could introduce tabs to toggle between SaaS and SM, but that's not worth it for something that will be consistent in 8.3.

Reply:
@wollefitz since I assumed you logged off for the day, I'll add another commit to the changes we are discussing to merge this. Feel free to open a new PR if you want any additional changes.

Reply:
Thanks a lot! I like the idea of tabs and will follow up with a PR in the near future.